### PR TITLE
Temporary fix for debian gcc-10, missing i386-cpuinfo.h

### PR DESCRIPTION
**Pull request by nuclearcat**
Builds with allmodconfig or when ggc plugins enabled failing, for example:

https://storage.kernelci.org/mainline/master/v5.17-rc7/x86_64/allmodconfig/gcc-10/logs/build-errors.log

Build error message example:
/usr/lib/gcc/x86_64-linux-gnu/10/plugin/include/config/i386/i386.h:2500:10: fatal error: common/config/i386/i386-cpuinfo.h: No such file or directory
/usr/lib/gcc/x86_64-linux-gnu/10/plugin/include/config/i386/i386.h:2500:10: fatal error: common/config/i386/i386-cpuinfo.h: No such file or directory

Using patch proposed in https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=980609#72

Signed-off-by: Denys Fedoryshchenko <denys.f@collabora.com>

### CODE REVIEW

**Comment by musamaanjum** [comment]
@nuclearcat Thank you. I'd found this bug and thought it'll get fixed at some point automatically when we'll get the next Debian release. This should unblock: https://github.com/kernelci/kernelci-core/issues/1041 I'll look at it once this PR is merged.

**Review by musamaanjum — APPROVED** [review]

**Comment by gctucker** [comment]
Starting an allmodconfig x86_64 build now on staging to confirm it's fixing the issue.

**Comment by nuclearcat** [comment]
Rebased on top of main and changed base

**Comment by gctucker** [comment]
The build worked fine:
https://storage.staging.kernelci.org/kernelci/staging-mainline/staging-mainline-20220315.0/x86_64/allmodconfig/gcc-10/steps.json

For some reason I can't find it on the staging web frontend, maybe the meta-data failed to get uploaded. The logs are on storage, there were no build errors. The only thing that failed is the dtb build, I guess the config is turned on by allmodconfig but it's not actually building for x86:
```
#
# 2022-03-15T20:56:27.618907
#
# make KBUILD_BUILD_USER=KernelCI ARCH=x86_64 HOSTCC=gcc CC="ccache gcc" O=/tmp/kci/linux/build -C/tmp/kci/linux -j34 dtbs
#
make: Entering directory '/tmp/kci/linux'
make[1]: Entering directory '/tmp/kci/linux/build'
make[1]: *** No rule to make target 'dtbs'.  Stop.
make[1]: Leaving directory '/tmp/kci/linux/build'
make: *** [Makefile:219: __sub-make] Error 2
make: Leaving directory '/tmp/kci/linux'
```

That's worth another GitHub issue to investigate how to avoid this error. Probably `CONFIG_OF_FLATTREE` should be removed from `allmodconfig`, we might add a fragment for that in KernelCI but ideally it should be fixed in the kernel sources.

**Review by gctucker — APPROVED** [review]

**Pull request merged**
